coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -183,8 +184,10 @@ def _compile_multi_match_regex(strings):
 
 def _extract_doc_comment_from_line(content, line, column, regex,
                                    marker_dict, language, docstyle):
-    begin_match = regex.search(content[line], column)
+    cur_line = content[line]
+    begin_match = regex.search(cur_line, column)
     if begin_match:
+        indent = SpacingHelper().get_indentation(cur_line)
```

Comment:
after a bit of thinking, I see a little problem: `SpacingHelper()` sets as an internal value 4 spaces as indentation. If this doesn't match the file indentation, we get a problem actually (the comment is then not correctly aligned). Maybe we should forget about `SpacingHelper` and get the indentation ourselves, maybe even not counting spaces and stuff, just taking the whitespaces inside the first line as reference.